webui/react/src/services/types.ts

```diff
@@ -27,7 +27,7 @@ interface ApiBase {
 // Designed for use with Swagger generated api bindings.
 export interface DetApi<Input, DetOutput, Output> extends ApiBase {
   postProcess: (response: DetOutput) => Output;
-  request: (params: Input) => Promise<DetOutput>;
+  request: (params: Input, options?: unknown) => Promise<DetOutput>;
```

Comment:
what options would go here instead of `params`

Reply:
we might want to include in as part of the params `Input & {options?: unknown}` just to make it easier to add future arguments this will make it sort of a keyword argument instead of a positional one

Reply:
same thing  as we did with fetch option

Reply:
My thinking here is that there are some other http options that we can potentially pass, even though we are only using `signal` at the moment. So this is to avoid any potential namespace pollution between params and options. Also to me, it seems a bit cleaner to call out the http options separately from params, so it's obvious what are API specific vs HTTP options.

Reply:
would we be using this for signal? It looks like you added this a while back. https://github.com/determined-ai/determined/blob/204d363985a0024c651785f0d49101d9f1f4e336/webui/react/src/services/utils.ts#L105